homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -56,34 +63,20 @@ async def async_setup_entry(
 
     entities: list[PlugwiseBinarySensorEntity] = []
     for device_id, device in coordinator.data.devices.items():
-        if device["class"] == "heater_central":
-            for description in BINARY_SENSORS:
-                if (
-                    "binary_sensors" not in device
-                    or description.key not in device["binary_sensors"]
-                ):
-                    continue
-
-                entities.append(
-                    PlugwiseBinarySensorEntity(
-                        coordinator,
-                        device_id,
-                        description,
-                    )
-                )
+        for description in BINARY_SENSORS:
+            if (
+                "binary_sensors" not in device
+                or description.key not in device["binary_sensors"]
+            ):
```

Comment:
We didn't have this check for the notify sensor before. Does it work for that sensor too?